lib/node_modules/@stdlib/stats/base/variancewd/lib/ndarray.js

```diff
@@ -62,48 +68,61 @@
 * -   Welford, B. P. 1962. "Note on a Method for Calculating Corrected Sums of Squares and Products." _Technometrics_ 4 (3). Taylor & Francis: 419–20. doi:[10.1080/00401706.1962.10490022](https://doi.org/10.1080/00401706.1962.10490022).
 * -   van Reeken, A. J. 1968. "Letters to the Editor: Dealing with Neely's Algorithms." _Communications of the ACM_ 11 (3): 149–50. doi:[10.1145/362929.362961](https://doi.org/10.1145/362929.362961).
 *
+*/
+
+/**
+* Computes the variance of a strided array using Welford's algorithm.
+*
 * @param {PositiveInteger} N - number of indexed elements
 * @param {number} correction - degrees of freedom adjustment
 * @param {NumericArray} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - stride length
+* @param {NonNegativeInteger} offsetX - starting index
 * @returns {number} variance
-*
-* @example
-* var floor = require( '@stdlib/math/base/special/floor' );
-*
-* var x = [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ];
-* var N = floor( x.length / 2 );
-*
-* var v = variancewd( N, 1, x, 2, 1 );
-* // returns 6.25
 */
-function variancewd( N, correction, x, stride, offset ) {
+function variancewd(N, correction, x, strideX, offsetX) {
+	var delta2;
 	var delta;
-	var mu;
 	var M2;
+	var mu;
 	var ix;
 	var v;
 	var n;
+	var o;
 	var i;
 
-	n = N - correction;
-	if ( N <= 0 || n <= 0.0 ) {
+	if (N <= 0) {
 		return NaN;
 	}
-	if ( N === 1 || stride === 0 ) {
+	if (N === 1 || strideX === 0) {
+		if (N - correction <= 0) {
+			return NaN;
+		}
 		return 0.0;
 	}
-	ix = offset;
+	n = N - correction;
+	if (n <= 0) {
+		return NaN;
+	}
+
+	o = arraylike2object(x);
+	if (o.accessorProtocol) {
+		return accessors(N, correction, o, strideX, offsetX);
+	}
+
+	ix = offsetX;
 	M2 = 0.0;
 	mu = 0.0;
-	for ( i = 0; i < N; i++ ) {
-		v = x[ ix ];
```

Comment:
This is wrong. L110 deals with accessor arrays. If `x` is not an accessor array, you should be indexing `x` directly.